libraries/provider_component_runit_supervisor_upstart.rb

```diff
@@ -4,11 +4,11 @@ class Chef
   class Provider
     class ComponentRunitSupervisor
       class Upstart < Chef::Provider::LWRPBase
-        provides :component_runit_supervisor, :platform_family => "debian"
         provides :component_runit_supervisor, :platform_family => "rhel" do |node|
           node['platform_version'].to_i == 6
         end
-        provides :component_runit_supervisor, :platform => %w[ amazon fedora ]
+        provides :component_runit_supervisor,
+          :platform => %w[ amazon fedora ubuntu ]
```

Comment:
This seems to be a behavior change that wasn't specified in the commit message. Was this just an oversight from when this lwrp was built? At the very least it seems like this should be in a separate commit from the debian change.